spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/vault/SpringVaultEnvironmentRepositoryFactory.java

```diff
@@ -39,16 +39,27 @@ public class SpringVaultEnvironmentRepositoryFactory
 
 	private final SpringVaultClientConfiguration clientConfiguration;
 
+	private SpringVaultTemplateBuilder vaultTemplateBuilder;
+
 	public SpringVaultEnvironmentRepositoryFactory(ObjectProvider<HttpServletRequest> request, EnvironmentWatch watch,
```

Comment:
I would mark this constructor as `@Deprecated` and have it call the new constructor with `null`